### ISSUE TITLE
[ BUG ] Test-FqlStatement doesn't like multiple values for a field/property

### ISSUE THREAD

**Issue by datorr2** [CrowdStrike/psfalcon#138]
**Describe the bug**
When specifying multiple values in an array for a filter, error is thrown:
```
Get-FalconDetection: Cannot validate argument on parameter 'Filter'. ''deadbeefdeadbeefdeadbeefdeadbeef'' is not a valid Falcon Query Language statement.
```

**To Reproduce**
```powershell
Get-FalconDetection -Filter "device.device_id:['deviceId1','deviceId2','deviceId3']"
```

**Expected behavior**
Expect FQL to pass validation and for command to run and return results.

**Environment (please complete the following information):**
 - OS: Windows 10 Enterprise (1909)
 - PowerShell: Core 7.1.5
 - PSFalcon: v2.1.5

**Additional context**
Tested the API using Swagger and works correctly there.

**Comment by datorr2** [comment]
![image](https://user-images.githubusercontent.com/28566413/140587909-a888dcbf-e234-471f-95a8-2198fddb2093.png)

Attached image of successful test in swagger.

**Comment by bk-cs** [comment]
Thank you for the report! My regex appears to be too strict. I will modify it and update with a fix.

**Comment by bk-cs** [comment]
I believe I have fixed this issue with the most recent commit. Will you please either:

* Download the source for v2.1.6 and import it

or

* Replace your local `Private\Private.ps1` file with the contents of this version: https://raw.githubusercontent.com/CrowdStrike/psfalcon/master/Private/Private.ps1
* Open a new PowerShell session
* Re-import your modified version of v2.1.5 with `Import-Module -Name PSFalcon -RequiredVersion 2.1.5`

And re-run your filtered `Get-FalconHost` search?

**Comment by bk-cs** [comment]
Resolved with v2.1.6 release.
